bcda/_site/index.html

```diff
@@ -0,0 +1,340 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+	<!-- Metadata for Twitter -->
+	<meta name="twitter:title" content="Beneficiary Claims API" />
```

Comment:
```suggestion
	<meta name="twitter:title" content="Beneficiary Claims Data API" />
```

Reply:
These should all be fixed.  In general we should avoid making changes in _site as they are built from the files in bcda-site-static.  It was fixed in the config.yml, but I failed to rebuild before committing.